browser.go

```diff
@@ -759,6 +759,66 @@ const (
 	MouseButtonMiddle = MouseButton3
 )
 
+type Joystick int
+
+const (
+	Joystick1  Joystick = 1
+	Joystick2  Joystick = 2
```

Comment:
Do we need to be explicit on those value?

Reply:
You’re right, `iota` will do fine. I’m adding a commit to the PR.

Reply:
Way better, thanks!